addons/ophthalmology/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -0,0 +1,26 @@
+class ACE_Medical_Treatment_Actions {
+    class Blink {
+        displayName = CSTRING(Blink_Display);
+        displayNameProgress = CSTRING(blink_action);
+        category = "bandage";
+        treatmentLocations = 0;
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 1;
+        medicRequired = 0;
+        treatmentTime = 2;
+        items[] = {};
+        condition = QUOTE(missionNamespace getVariable [ARR_2(QQGVAR(enable),true)]);
```

Comment:
we can use GETMVAR(MyVarName,objNull) macro

Reply:
We have no quoted getmvar because we need to quote the statement, but I've added it